pkg/stanza/operator/input/udp/udp.go

```diff
@@ -151,6 +155,14 @@ type Input struct {
 	encoding  encoding.Encoding
 	splitFunc bufio.SplitFunc
 	resolver  *helper.IPResolver
+
+	messageQueue chan MessageAndAddr
+	stopOnce     sync.Once
+}
+
+type MessageAndAddr struct {
```

Comment:
Does this struct need to be exported? If not, prefer not to in order to minimize package surface area.